### DISCUSSION TITLE
Lower Oxygen Limit

### DISCUSSION THREAD

**Discussion by LucilePl** [firemodels/cfast#2115] [Q&A]
Hello,
I would like to check that my HRR decreases well from the moment the lower oxygen limit is reached, but I do not know what mass fraction CFAST takes into account. Is it an average or the mass fraction of oxygen in the hot layer? Moreover CFAST provides the molar fraction of oxygen, so to calculate the mass fraction of oxygen I need to know the molar mass of soot taken by CFAST, but I do not find this value. I take any information on that.

Thank you in advance!

**Comment by preneke** [comment]
If you look in the _compartments.csv we convert the OD to mg/m^3 by multiplying od by 264.6903. The volume of the upper layer is also in the same spreadsheet.

**Reply by LucilePl** [reply]
I don’t understand what you’re getting at, what OD are we talking about? What does the value of 264.6903 refer to?

**Comment by drjfloyd** [comment]
Per 3.2 in the CFAST Technical Reference CFAST breaks the HRR calculation into two parts. First it determines how much fuel can burn in the lower layer based on the lower layer gas entrained from the fire height to the bottom of the upper layer. This uses the O2 concentration in the lower layer aobve the lower limit.  Second, if any unburned fuel remains, then CFAST looks at the total mass of oxygen above the limit in the upper layer.

The MW of SOOT is taken as 12. With a typical 15 % lower limit, even with a soot and CO yield of 20 %, soot would be less than a 1 % mole fraction. It doesn't have much impact on the concentration of oxygen.
